mscore/qml/palettes/Palette.qml

```diff
@@ -275,7 +275,9 @@ GridView {
 //                             ToolTip.timeout: Qt.styleHints.mousePressAndHoldInterval
             ToolTip.text: model.toolTip
 
-            text: model.toolTip
+            text: model.accessibleText
+            Accessible.name: model.accessibleText
```

Comment:
If only `text` field is actually used by screen readers perhaps there is no need to set `Accessible.name` too. I am not sure why `Accessible.name` does not work though, my guess was that it should take a priority over `text` but perhaps it does not work so in Qt Quick.

Reply:
It works with or without Accessible.name indeed.  But the documentation is confusing to me.  Both say they might be used for accessibility purposes.  So I figure, no harm in setting them both.  And Accessible.description too.  I hate to settle for something just because it works with one particular version of Qt, one particular version of NVDA, I'd rather put the information out there anywhere a screenreader might read it.